sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -1227,7 +1227,11 @@ private ServiceBusSessionReceiverClientBuilder() {
          * abandoned}.
          *
          * @return The modified {@link ServiceBusSessionReceiverClientBuilder} object.
+         *
+         * @deprecated Auto complete will be removed on the session receiver client
+         * When using the bare session receiver client, users will have to determine when a message can be completed (settled).
          */
+        @Deprecated(since = "7.8.0", forRemoval = true)
```

Comment:
`forRemoval` is Java 9+, need to remove it for compilation to pass

Reply:
This file was not supposed to be picked up :)